Planner/src/TaskDAO.java

```diff
@@ -49,20 +49,15 @@
 	            }
 	            int assignedTo = rset.getInt("assignedTo");
 	            int taskID = rset.getInt("taskID");
-	            /*
-	            // Step 3 & 4: Issue a SELECT to check the UPDATE.
-				String selectUser = "select * from users" + " where ID=" + String.valueOf(assignedTo);
-				System.out.println("The SQL query is: " + strSelect);  // Echo for debugging
-				ResultSet rsetUser = stmt.executeQuery(selectUser);
-				User user = null;
-				while(rsetUser.next()) {   // Move the cursor to the next row
-					String name = rsetUser.getString("firstName");
-		            String surname = rsetUser.getString("surname");
-		            int id = rsetUser.getInt("ID");
-		            user = new User(name, surname, id);
-				}
-				*/
-	            Task task = new Task(nameTask, descriptionTask, startTime, endTime, null, taskID);
+	            
+	            // you need to replace this commented code with a for loop in the users array. You need to search the user that has the id == assignedTo and put this user in the constructor at line 65 (56). Do the same thing for insert and update methods.
+	            for (User user : users) {
+	            	int selectedID = user.getId();
+	            	selectedID= assignedTo;
```

Comment:
Here we don't need to place the value of assignedTo in selectedId. We just need to check that selectedID is equal to assigned to -> so if (selectedID == assignedTo) . If they are equal we can put the break instruction. The break instruction will make the for loop stop executing and it will exit from the for loop. We don't need to do the whole for once we found the User that we are searching for. 
`
User assignedUser = null;
for (User user : users) {

      int selectedID = user.getId();

     if (selectedID == assignedTo) {

         assignedUser = user;

         break;

    }

Task task = new Task(nameTask, descriptionTask, startTime, endTime, assignedUser, taskID);

`